pro/src/styles/components/layout/inputs/RadioInput/_RadioInput.scss

```
@@ -73,5 +73,9 @@ input[type="radio"] {
 
   .input-radio-input {
     align-self: flex-start;
+
+    &.error:not(:focus-within) {
+      border: 2px solid $red-error;
```

Si tout vas bien on vas retrouver nos pre-commit d'ici peux, et les oublie de `rem` seront tagé en amont !

```suggestion
      border: rem(2px) solid $red-error;
```